src/app/(nousSoutenir)/(don)/donation/page.js

```
@@ -218,51 +229,137 @@ export default function Donation() {
+    }
+
+    try {
+      // Étape 1 : Envoyer une requête au backend pour générer le paiement
+      const response = await fetch('http://127.0.0.1:5001/bab-rayan-b04a0/us-central1/api/v1/cmi/createCmi', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          fullname: userData && userData.fullName ? userData.fullName : "Anonyme",
+          email: userData && userData.email ? userData.email : "Anonyme@gmail.com",
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)